docs/issue_triage_guide.md

```diff
@@ -0,0 +1,35 @@
+# Issue Triage Guide
+
+This guide describes how to triage GitHub issues on the [issue queue](https://github.com/openservicemesh/osm/issues) for Open Service Mesh (OSM).
+
+## Steps for Triage
+1. Identify the cause of the issue as explained [below](#identifying-the-cause-of-the-issue)
+1. Tag the issue with relevant tags
```

Comment:
I'd add a little more info here.
`Relevant labels would include what area of the codebase the issue is most aligned with (ex. cli) or if it is a bug, label the issue with the bug label.`

Let's also use the term "label" instead of tags so we're as explicit as possible about the steps.